webapp/channels/src/components/post_view/channel_intro_message/channel_intro_message.tsx

```diff
@@ -30,6 +32,8 @@ import * as Utils from 'utils/utils';
 import AddMembersButton from './add_members_button';
 import PluggableIntroButtons from './pluggable_intro_buttons';
 
+import '../../../sass/components/_action-button.scss';
```

Comment:
@larkox Looks like this file is not imported (or tied to the base sass file). So it won't be imported automatically I think.
But we can probably should import it in the main sass file and remove it from here.

If we feel like we should not include it globally, then this file should be included in the channel_intro_message folder with the other files and imported directly.

Reply:
@larkox @asaadmahmood I moved this to the `_module.scss` file instead in the latest commit.

Reply:
Yep, Asaad has that right. I would consider trying to import CSS files where they're used to assist with code splitting, but that only works for more narrow classes which are closely associated to a React component. Since `action-button` doesn't have an associated component yet, adding it to the root CSS file works